task6_Sychev_Alexander/include/calc_distance_handling.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <cmath>
+#include <map>
+#include <functional>
+#include <stdexcept>
+
+
+static class Calc_distance_handling
```

Comment:
you are creating only for doing calculations!, you can use a map with lambda expression and save the functions there